README.md

```diff
@@ -453,6 +454,14 @@ CSV
 
 * [cl-csv](https://github.com/AccelerationNet/cl-csv) - A library for parsing CSV files. [3-clause BSD][15].
 
+Plotting
+--------
+
+* [vgplot](https://github.com/volkers/vgplot) - an interface to the
+  gnuplot plotting utility with the intention to resemble to some of
```

Comment:
I'm not a native speaker, but I suppose the second `to` is too much. 

`...with the intention to resemble some of...`